### ISSUE TITLE
In functions : "Set text return value", return number

### ISSUE THREAD

**Issue by Bouh** [4ian/GDevelop#1040]
## Describe the bug
The event return text in functions ask number or expression, but it can't return text.
I've tryed to return "text" or a variable string but nothing is returned correctly. 
Value returned is always 0

![image](https://user-images.githubusercontent.com/1670670/56849682-d6b66800-68f7-11e9-91bb-b73c80b0b65d.png)

Look the function **ReturnLineWhereIsTheValue**, the value returned go to my awesome variable called **maSortieQuiFonctionnePas**
Example project  : 
[My project.zip](https://github.com/4ian/GDevelop/files/3123856/My.project.zip)
(You need copy the file _subtitles_en.vtt_ on your desktop or change the path)

## Other details
B66, W10

**Comment by 4ian** [comment]
I'll take a look and see if I can release a version with a few bug fixes

**Comment by 4ian** [comment]
Should be fixed in 668bd6a9831293db6c759dbd99b218fd1be1bbef

If you have the development version, re-run `npm start` (well, in ~30minutes, the time for the updated libGD.js to be compiled) and you should get the fix :)

**Comment by Bouh** [comment]
I'm having some problems with the dev version
Like if the libGD.js is not the last build.
- When i save 
![image](https://user-images.githubusercontent.com/1670670/56895134-64c15880-6a88-11e9-8919-81095ba32b00.png)


```
Microsoft Windows [version 10.0.17763.437]
(c) 2018 Microsoft Corporation. Tous droits réservés.

C:\GDevelop\newIDE\app>npm install

> grpc@1.10.1 install C:\GDevelop\newIDE\app\node_modules\grpc
> node-pre-gyp install --fallback-to-build --library=static_library

[grpc] Success: "C:\GDevelop\newIDE\app\node_modules\grpc\src\node\extension_binary\node-v57-win32-x64-unknown\grpc_node.node" is installed via remote

> gdevelop-ide@0.1.0 postinstall C:\GDevelop\newIDE\app
> npm run import-resources


> gdevelop-ide@0.1.0 import-resources C:\GDevelop\newIDE\app
> cd scripts && node import-libGD.js && node import-res-folder.js && node import-GDJS-Runtime.js && node import-monaco-editor.js && node import-zipped-editor.js piskel 5.0.0-beta56 && node import-zipped-editor.js jfxr 5.0.0-beta55

🌐 Downloading pre-built libGD.js from https://s3.amazonaws.com/gdevelop-gdevelop.js (be patient)...
ℹ️ Trying to download libGD.js for HEAD.
⚠️ Can't download libGD.js (can't find hash=85b788b9c5e82e4def253152c8904a750cd17613).
ℹ️ Maybe libGD.js was not automatically built yet, try again in a few minutes.
ℹ️ Trying to download libGD.js for HEAD~1.
⚠️ Can't download libGD.js (can't find hash=75140265d6f8bd964cdbc03143a706dc3da7d70b).
ℹ️ Maybe libGD.js was not automatically built yet, try again in a few minutes.
ℹ️ Trying to download libGD.js for HEAD~2.
✅ libGD.js downloaded and stored in public/libGD.js
✅ Copied libGD.js to node_modules folder
✅ Images from res, CppPlatform and JsPlatform properly copied in public folder
Copying GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\resources\GDJS\Runtime"...
0 fichier(s) copi�(s)
0 fichier(s) copi�(s)
✅ Copied GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\resources\GDJS\Runtime".
Copying GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\node_modules\GDJS-for-web-app-only\Runtime"...
65 fichier(s) copi�(s)
79 fichier(s) copi�(s)
✅ Copied GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\node_modules\GDJS-for-web-app-only\Runtime".
✅ Sources of Monaco Editor properly copied in public folder
✅ piskel-editor already existing in public/external/piskel folder - skipping download
✅ jfxr-editor already existing in public/external/jfxr folder - skipping download
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules\fork-ts-checker-webpack-plugin\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 2692 packages from 1075 contributors and audited 52998 packages in 53.728s
found 575 vulnerabilities (290 low, 247 moderate, 37 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details

C:\GDevelop\newIDE\app>npm start

> gdevelop-ide@0.1.0 start C:\GDevelop\newIDE\app
> npm run import-resources && react-scripts start


> gdevelop-ide@0.1.0 import-resources C:\GDevelop\newIDE\app
> cd scripts && node import-libGD.js && node import-res-folder.js && node import-GDJS-Runtime.js && node import-monaco-editor.js && node import-zipped-editor.js piskel 5.0.0-beta56 && node import-zipped-editor.js jfxr 5.0.0-beta55

🌐 Downloading pre-built libGD.js from https://s3.amazonaws.com/gdevelop-gdevelop.js (be patient)...
ℹ️ Trying to download libGD.js for HEAD.
⚠️ Can't download libGD.js (can't find hash=85b788b9c5e82e4def253152c8904a750cd17613).
ℹ️ Maybe libGD.js was not automatically built yet, try again in a few minutes.
ℹ️ Trying to download libGD.js for HEAD~1.
⚠️ Can't download libGD.js (can't find hash=75140265d6f8bd964cdbc03143a706dc3da7d70b).
ℹ️ Maybe libGD.js was not automatically built yet, try again in a few minutes.
ℹ️ Trying to download libGD.js for HEAD~2.
✅ libGD.js downloaded and stored in public/libGD.js
✅ Copied libGD.js to node_modules folder
✅ Images from res, CppPlatform and JsPlatform properly copied in public folder
Copying GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\resources\GDJS\Runtime"...
0 fichier(s) copi�(s)
0 fichier(s) copi�(s)
✅ Copied GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\resources\GDJS\Runtime".
Copying GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\node_modules\GDJS-for-web-app-only\Runtime"...
0 fichier(s) copi�(s)
0 fichier(s) copi�(s)
✅ Copied GDJS and extensions runtime files (*.js) to "C:\GDevelop\newIDE\app\node_modules\GDJS-for-web-app-only\Runtime".
✅ Sources of Monaco Editor properly copied in public folder
✅ piskel-editor already existing in public/external/piskel folder - skipping download
✅ jfxr-editor already existing in public/external/jfxr folder - skipping download
Starting the development server...
Compiled successfully!

You can now view gdevelop-ide in the browser.

  Local:            http://localhost:3000/
  On Your Network:  http://192.168.0.32:3000/

Note that the development build is not optimized.
To create a production build, use yarn build.

Terminer le programme de commandes (O/N) ?
^C
C:\GDevelop\newIDE\app>


```

**Comment by 4ian** [comment]
In your logs, I see that `HEAD~1` (which means "the commit before the last commit of the branch") has hash `75140265d6f8bd964cdbc03143a706dc3da7d70b`, which is old (well, a few days old). I just updated the way libGD.js is built a few days ago so that it's built automatically for each commit on master.
So you should pull master branch :)

The crash you're having is actually solved 3 days ago in 2cf2622b9e02478dcb4ebac8924a6ef5e759b568

(For your interest, if you look at the list of commits, you can see the checks:
![image](https://user-images.githubusercontent.com/1280130/56897626-eca66300-6a86-11e9-8da2-5ec71a1232f8.png)

If the `continuous-integration/travis-ci` check is green, it's that libGD.js was built and is ready for this commit. But again this is only the case for the recent commits. Old commits created before won't have any libGD.js built for them)
